### ISSUE TITLE
CRD for micro?

### ISSUE THREAD

**Issue by asim** [magaldima/macro#1]
Is this a micro CRD for k8s? Looks quite interesting. Would like to know more.

**Comment by magaldima** [comment]
@asim yes.. it's still very much a WIP and I don't have a lot of bandwidth.. so working on this in some free time. This was inspired by:  [micro/kubernetes#16](https://github.com/micro/kubernetes/issues/16)

**Comment by asim** [comment]
Awesome, well very much appreciated and very interested in where this goes. If I had funding I would hire you full time :)
